### ISSUE TITLE
Peer dependency issue with npm3. Mongoskin needs to be updated.

### ISSUE THREAD

**Issue by manikandants** [agenda/agenda#201]
```
npm WARN EPEERINVALID mongoskin@1.4.13 requires a peer of mongodb@~1.4 but none was installed.
npm ERR! Linux 3.19.0-28-generic
npm ERR! argv "node" "/usr/bin/npm" "install"
npm ERR! node v0.12.7
npm ERR! npm  v3.3.3
npm ERR! path /home/user/project/node_modules/agenda/node_modules/mongodb/node_modules/readable-stream/node_modules/core-util-is
npm ERR! code ENOENT
npm ERR! errno -2

npm ERR! enoent ENOENT, rename '/home/user/project/node_modules/agenda/node_modules/mongodb/node_modules/readable-stream/node_modules/core-util-is'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /home/user/project/npm-debug.log
```

**Comment by mahnunchik** [comment]
+1

**Comment by manikandants** [comment]
:+1:

**Comment by nha** [comment]
Same with `npm 3.3.6` on osx.

npm WARN EPEERINVALID mongoskin@1.4.13 requires a peer of mongodb@~1.4 but none was installed.

What is the meaning of this warning ?

**Comment by wesley-harding** [comment]
:+1:

**Comment by loris** [comment]
Can be closed, mongoskin has been replace in `0.7.0`
